src/media/js/addon/components/reviewActions.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import * as constants from '../constants';
+import ConfirmButton from '../../site/components/confirmButton';
+
+
+export default class ReviewActions extends React.Component {
+  static propTypes = {
+    isProcessing: React.PropTypes.bool,
+    reject: React.PropTypes.func.isRequired,
+    publish: React.PropTypes.func.isRequired,
+    status: React.PropTypes.string.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isShowingMessageBox: false,
```

Comment:
then there'd be a bunch of message boxes visible for each version without much context as to what they do